cmd/addoperator.go

```diff
@@ -87,7 +88,21 @@ func (p *AddOperatorParams) PreInteractive(ctx ActionCtx) error {
 		return err
 	}
 	if ok {
-		p.jwtPath, err = cli.Prompt("path or url for operator jwt", p.jwtPath, true, cli.PathOrURLValidator())
+		p.jwtPath, err = cli.Prompt("path or url for operator jwt", p.jwtPath, true, func(v string) error {
+			// is it is an URL or path
+			pv := cli.PathOrURLValidator()
+			if err := pv(v); err != nil {
+				// if it doesn't exist - could it be the name of well known operator
+				if os.IsNotExist(err) {
+					wko, _ := FindKnownOperator(v)
+					if wko != nil {
+						return nil
+					}
+					return nil
+				}
+			}
+			return nil
```

Comment:
So we never return an error in this validation function..

Reply:
yes this last return is wrong, it needs to return err at that point.